cli/tests/testdata/f1.yaml

```diff
@@ -23,7 +23,8 @@ timescaledb-single:
 #   https://github.com/timescale/promscale/tree/master/helm-chart
 promscale:
   enabled: true
-  image: timescale/promscale:latest
+  image: vineeth97/promscale:backfill-latest
+  replicaCount: 1
```

Comment:
wait isn't f1 supposed to be the ha test?

Reply:
yup, it's an HA test. This image is backfill latest HA PR build. Ante PR fixes metric exposition nit on startup.